packages/orbit-design-tokens/src/dictionary/formatters/javascriptThemeableObject.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+const _ = require("lodash");
+
+const fileTemplate = require("./fileTemplate");
+const withFlowComment = require("./withFlowComment");
+
+const fnTemplate = tokens => {
+  return `export default function createTheme(theme) {\n  return {\n${tokens}\n  };\n}\n`;
+};
+const indent = "  ";
+const createVariable = (name, value) => `${indent}${indent}${name}: ${value},`;
```

Comment:
I'm currently doing another formatter – we need also `.js.flow` and `.d.ts` files, I will unify it once it will be done. It would be hard the opposite way 😄